src/session_impl.cpp

```diff
@@ -5699,9 +5699,8 @@ namespace {
 		}
 
 #ifndef TORRENT_DISABLE_LOGGING
-		session_log("starting DHT, running: %s, router lookups: %d, aborting: %s"
```

Comment:
I'm not sure about this. it makes the log line look a bit deceitful.

Reply:
revert?

Reply:
I think that would be best